code/obj/artifacts/artifact_items/pitcher.dm

```diff
@@ -66,7 +66,7 @@
 		if (prob(3))
 			reagents.add_reagent("liquid spacetime", 25)
 		if (prob(1))
-			reagents.add_reagent("fuzz", 5)
+			reagents.add_reagent("rat_venom", 5)
```

Comment:
Do we even still want this in pitchers, considering it's one of like 6 or 7 chems that are blacklisted from the random chem list?

Reply:
Excellent question, dunno!